src/components/TagInfoPage.tsx

```diff
@@ -0,0 +1,54 @@
+import { TagInfo } from '@equinor/procosys-webapp-components';
```

Comment:
Kunne du endret navn til TagInfoWrapper i stedet for TagInfoPage? Det er jo PunchPage eller ChecklistPage som er hele siden, det som nå heter TagInfoPage er kun innholdet på siden. 
Det var sikkert jeg som sa hva du skulle kalle det, men det er bedre om den heter wrapper. Hvis du ser i Checklist mappen så heter wrapperen rundt selve checklist komponenten ChecklistWrapper